### PR TITLE
Let operator manage autoscaling of node groups

### PR DESCRIPTION
**Pull request by katexochen**
### Proposed change(s)

- Let operator manage autoscaling of node groups
- Remove autoscaling from CLI
- Update manual workflow for operator building to not require a tag

### Additional info
- Operator image version for testing: [v0.0.1-0.20220919130935-784ae5f339a1](https://github.com/orgs/edgelesssys/packages/container/constellation%2Fnode-operator/42285670?tag=v0.0.1-0.20220919130935-784ae5f339a1)

### Checklist

- [x] Update [CHANGELOG.md](https://github.com/edgelesssys/constellation/blob/main/CHANGELOG.md)
- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [ ] Link to Milestone

### CODE REVIEW

**Comment by katexochen** [comment]
Still, working on the documentation, but if you like you can start reviewing the code @malt3 .

File: operators/constellation-node-operator/api/v1alpha1/scalinggroup_types.go [hunk 1]
```diff
@@ -21,10 +26,22 @@ type ScalingGroupSpec struct {
...
 	// GroupID is the CSP specific, canonical identifier of a scaling group.
 	GroupID string `json:"groupId,omitempty"`
+	// AutoscalerGroupName is name that is expected by the autoscaler.
+	AutoscalerGroupName string `json:"autoscalerGroupName,omitempty"`
 	// Autoscaling specifies wether the scaling group should automatically scale using the cluster-autoscaler.
 	Autoscaling bool `json:"autoscaling,omitempty"`
+	// Min is the minimum number of nodes in the scaling group.
+	Min int32 `json:"min,omitempty"`
+	// Max is the maximum number of nodes in the scaling group.
+	Max int32 `json:"max,omitempty"`
```
Comment (malt3) [inline operators/constellation-node-operator/api/v1alpha1/scalinggroup_types.go:36]:
```suggestion
	// Min is the minimum number of nodes in the scaling group (used by cluster-autoscaler).
	Min int32 `json:"min,omitempty"`
	// Max is the maximum number of autoscaled nodes in the scaling group (used by cluster-autoscaler).
	Max int32 `json:"max,omitempty"`
```

This or similar description to distinguish the from actual min / max in scaling groups.

**Review by malt3 — COMMENTED** [review]
Code LGTM. Only nits. 
Will do some testing and get back to you.

**Review by malt3 — APPROVED** [review]
LGTM. Only minor changes.

**Pull request merged**